sentry/src/main/java/io/sentry/SentryTracer.java

```diff
@@ -384,6 +385,8 @@ private ISpan createChild(
               }
             });
     span.setDescription(description);
+    span.setData(
```

Comment:
Did we agree to use the same `blocked_main_thread` key? I'd rather us choose something different, because we don't know for sure if it blocked the main thread, we know that it was just started there? Also, if we pick a different key, the current behavior for File I/O or DB spans will remain the same with `blocked_main_thread`.

Reply:
Nope, this wasn't discussed yet - thanks for bringing this up. @shruthilayaj I think Roman has some good points here, would it make sense for you as well to have a separate flag for main thread flagging?

Reply:
Yes, it makes sense to have these as separate flags!

Reply:
👍 
just to confirm, sending `thread.id` and `thread.name` in addition to `started_on_main_thread` flag right?

Reply:
mm, since in the RFC we say it's the thread the span originated from, I think `thread.name: main` would be the same thing pretty much, so we don't need both flags?
![image](https://github.com/getsentry/sentry-java/assets/4999776/c984af92-6802-49d1-b3f5-df229b9dde6a)

Reply:
yeah makes sense, and will thread name always be standardized to `main` if it started on the main thread regardless of the SDK it's coming from?

Reply:
I'd start with just `main`, I guess we can expand it later. We can ensure that it's called `main` from the SDKs side